onadata/libs/serializers/project_serializer.py

```diff
@@ -415,6 +415,17 @@ def update(self, instance, validated_data):
                 members_team = get_organization_members_team(owner.profile)
                 OwnerRole.add(owners_team, instance)
                 ReadOnlyRole.add(members_team, instance)
```

Comment:
Should users have `ReadOnlyRole` by default when a project is transferred to an organization  @ivermac @ukanga ?

Reply:
Yes.